app.js

```diff
@@ -232,6 +234,8 @@ function getTickInterval() {
 function setTickInterval(newTickInterval) {
   const localStorage = window.localStorage;
   localStorage.setItem('tickInterval', newTickInterval);
+  // ^ this obv works (may not in weird Private Browsing Modes), but you could also add global variables instead
+  // then close off the namespace a bit ...
 }
```

Comment:
I think the only global vars in your code are the buttons which does make the app.js file nice and clean looking. But I think one option would be to change some localStorage items to JS variables. 

Then later when you're breaking it up into ng components (or node modules), you could have them all closed off nice and secure in their own namespace (like the end of the udemy 'weird parts' video on jquery)